src/Components/MainSection/MainSection.jsx

```diff
@@ -0,0 +1,72 @@
+import "./mainsection.css";
+
+export function Main() {
+
+    const categories = ["https://res.cloudinary.com/doo5jbomi/image/upload/v1648929327/Assets%20For%20Ecom/Brand%20Categories/brand-light-logo12_hou2tm.png",
```

Comment:
These URLs can be put inside a `constants` or `utils` file and imported here. This will make the components clean.